samwise.js

```diff
@@ -7,10 +7,10 @@
 /* global clearTimeout, setTimeout, document */
```

Comment:
SCRIPT-You did a great job handling all the possible error states in the UI. It enhances the user experience significantly.